integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/l0promotion/EnvironmentPrivilegedUserTest.java

```diff
@@ -0,0 +1,133 @@
+            and = "execution should pass in case of assigned EnvironmentPrivilegedUser role but changing the user to root should fail"
+    )
+    public void testSudoCommands(TestContext testContext) {
+        testContext
+                .given(UmsTestDto.class)
+                .assignTarget(EnvironmentTestDto.class.getSimpleName())
+                .withEnvironmentAdmin()
+                .when(umsTestClient.assignResourceRole(L0UserKeys.USER_ENV_CREATOR, regionAwareInternalCrnGeneratorFactory))
+                .validate();
+
+        String workloadUsernameEnvCreator = testContext.getRealUmsUserByKey(L0UserKeys.USER_ENV_CREATOR).getWorkloadUserName();
+
+        useRealUmsUser(testContext, L0UserKeys.USER_ENV_CREATOR);
+
+        testContext
+                .given(UmsTestDto.class).assignTarget(EnvironmentTestDto.class.getSimpleName())
+                .when(umsTestClient.setWorkloadPassword(WORKLOAD_PASSWORD, regionAwareInternalCrnGeneratorFactory))
+                .given(FreeIpaUserSyncTestDto.class)
+                .when(freeIpaTestClient.syncAll())
+                .await(OperationState.COMPLETED)
+                .given(FreeIpaTestDto.class)
+                .when(freeIpaTestClient.describe())
+                .thenException((tc, testDto, client) -> {
+                    sshSudoCommandActions.executeCommand(getIpAddresses(tc), workloadUsernameEnvCreator, WORKLOAD_PASSWORD, LIST_RULES_FLAG);
+                    return testDto;
+                }, TestFailException.class, expectedMessage("sudo command failed on '.*' for user '" + workloadUsernameEnvCreator + "'."))
+                .given(UmsTestDto.class)
+                .withEnvironmentPrivilegedUser()
+                .when(umsTestClient.assignResourceRole(L0UserKeys.USER_ENV_CREATOR, regionAwareInternalCrnGeneratorFactory))
+                .given(FreeIpaUserSyncTestDto.class)
+                .when(freeIpaTestClient.syncAll())
+                .await(OperationState.COMPLETED)
+                .then((tc, testDto, client) -> {
+                    Set<String> ipAddresses = getIpAddresses(tc);
+                    sshSudoCommandActions.executeCommand(ipAddresses, null, null, SSSD_RESTART, SSSD_STATUS);
+                    tc.waitingFor(Duration.ofMinutes(2), "Waiting for SSSD to be synchronized has been interrupted");
+                    sshSudoCommandActions.executeCommand(ipAddresses, workloadUsernameEnvCreator, WORKLOAD_PASSWORD, LIST_RULES_FLAG);
+                    return testDto;
+                })
+                .thenException((tc, testDto, client) -> {
+                    sshSudoCommandActions.executeCommand(getIpAddresses(tc), workloadUsernameEnvCreator, WORKLOAD_PASSWORD, CHANGE_USER_TO_ROOT_COMMAND);
+                    return testDto;
+                }, TestFailException.class, expectedMessage("sudo command failed on '.*' for user '" + workloadUsernameEnvCreator + "'."))
+                .validate();
+    }
```

Comment:
Here you should introduce:
```
 useRealUmsUser(testContext, L0UserKeys.USER_ACCOUNT_ADMIN);
```
because of in the end of the test 
```
com.sequenceiq.it.cloudbreak.exception.TestFailException: Default tag: [owner] value: [cb-qe-machine-envcreator-a] NOT equals [cb-qe-a-envcreator@cloudera.com] acting user name! 
	at com.sequenceiq.it.util.TagsUtil.validateOwnerTag(TagsUtil.java:141)
	at com.sequenceiq.it.util.TagsUtil.lambda$verifyTags$1(TagsUtil.java:90)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at com.sequenceiq.it.util.TagsUtil.verifyTags(TagsUtil.java:87)
	at com.sequenceiq.it.util.TagsUtil.verifyTags(TagsUtil.java:70)
	at com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest.lambda$tearDownSpotValidateTags$0(AbstractE2ETest.java:51)
	at java.base/java.util.LinkedHashMap$LinkedValues.forEach(LinkedHashMap.java:647)
	at com.sequenceiq.it.cloudbreak.testcase.e2e.AbstractE2ETest.tearDownSpotValidateTags(AbstractE2ETest.java:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
	at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
	at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
	at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
	at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:677)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:661)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```
So this is producing configuration error:
```
===============================================
ssh-tests
Total tests run: 1, Passes: 1, Failures: 0, Skips: 0
Configuration Failures: 1, Skips: 1
===============================================
```

Reply:
Thanks, done.